django_atlassian/models/djira.py

```diff
@@ -41,6 +48,37 @@ def create_from_json(self, json):
         return self.model(**args)
 
 
+class JiraManagerMixin(JIRA):
+    def __init__(self, *args, **kwargs):
+        options = copy.copy(JIRA.DEFAULT_OPTIONS)
+        try:
+            db_alias = router.db_for_read(self.model)
+            db_settings = connections.databases[db_alias]
+            if db_settings['ENGINE'] == 'django_atlassian.backends.jira':
+                conn = connections[db_alias].connection
+                if conn:
+                    server = conn.uri
+                    user = conn.user
+                    password = conn.password
+                    super(JiraManagerMixin, self).__init__(
+                        server=server,
+                        basic_auth=(user, password))
+        except Exception as err:
+            logger.error(err)
+
+
+class JiraAtlassianManager(EmptyManager, JiraManagerMixin):
```

Comment:
We don't need to the Atlassian prefix right? on the issue case it is named as "JiraIssueManager", here "JiraAtlassianManager"